tests/mcuboot/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = hello_mcuboot
+
+BOARD ?= nrf52dk
+
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := nrf52dk
+
+export IMAGE_VERSION = 1.1.1+1
```

Comment:
- isn't the image always aligned on a flash page boundary?
- isn't the IMG_HDR always 512 for mcuboot?